regression/verilog/system-functions/countones2.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+countones2.sv
+--bound 10
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
```

Comment:
Why is this (still) a `KNOWNBUG`, i.e., what aspect of this test is unexpectedly not working?

Reply:
Now works!